RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -95,39 +95,23 @@
 from RecoMET.METFilters.hfNoisyHitsFilter_cfi import *
 
 metFilters = cms.Sequence(
-   HBHENoiseFilterResultProducer *
-   HBHENoiseFilter *
-   primaryVertexFilter*
-#   HBHENoiseIsoFilter*
-#   HcalStripHaloFilter *
-   CSCTightHaloFilter *
-#   hcalLaserEventFilter *
-   #Various proposals for updated halo filters.
-   ##2015 proposals: 
-   #CSCTightHaloTrkMuUnvetoFilter *
-   #CSCTightHalo2015Filter *
-   ##2016 proposals
-   #globalTightHalo2016Filter*
-   #globalSuperTightHalo2016Filter*
-   EcalDeadCellTriggerPrimitiveFilter* 
-   ecalBadCalibFilter*
-#   *goodVertices * trackingFailureFilter *
-   eeBadScFilter*
-#   ecalLaserCorrFilter *
-#   trkPOGFilters
-   chargedHadronTrackResolutionFilter *
-   BadChargedCandidateFilter*
-   BadPFMuonFilter *
-   BadPFMuonDzFilter *
-   hfNoisyHitsFilter *
-   BadChargedCandidateSummer16Filter*
-   BadPFMuonSummer16Filter *
-   muonBadTrackFilter
+    goodVertices *
+    globalSuperTightHalo2016Filter *
+    HBHENoiseFilterResultProducer  *
+    HBHENoiseFilter *
+    HBHENoiseIsoFilter *
+    EcalDeadCellTriggerPrimitiveFilter *
+    BadPFMuonFilter *
+    BadPFMuonDzFilter *
+    EcalDeadCellTriggerPrimitiveFilter * 
+    ecalBadCalibFilter *
```

Comment:
According to the [twiki](https://twiki.cern.ch/twiki/bin/viewauth/CMS/MissingETOptionalFiltersRun2#Analysis_Recommendations_for_ana) you point out, this is not among the recommended filters for 2016 data and MC (UL): do you plan to remove it with a modifier?